distributed/src/main/scala/asobu/distributed/protocol/EndPointDefinition.scala

```diff
@@ -0,0 +1,100 @@
+package asobu.distributed.protocol
+
+import akka.actor.ActorPath
+import asobu.distributed.RequestEnricherDefinition
+import play.routes.compiler._
+import EndpointDefinition._
+
+@SerialVersionUID(1L)
+case class EndpointDefinition(
+  prefix: Prefix,
+  verb: Verb,
+  path: Seq[PathPart],
+  call: String,
+  handlerAddress: HandlerAddress,
+  clusterRole: String,
+  enricherDef: Option[RequestEnricherDefinition] = None,
+  version: Option[Int] = None
+)
+
+@SerialVersionUID(1L)
+case class Verb(value: String)
+
+sealed trait PathPart
```

Comment:
Are these just copies of what is in play.routing?  Can we just use those, since we already have that dependency, or is there some issue with doing that?